samples/tab-channel-context/nodejs/ClientApp/package.json

```diff
@@ -42,10 +42,5 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  },
-  "proxy": "http://localhost:3978",
-  "devDependencies": {
-    "@types/react": "^17.0.20",
-    "@types/react-router-dom": "^5.3.0"
   }
```

Comment:
Why we removed it?

Reply:
there is not server call so we don't need proxy and also we not required dev dependencies.